ccpp/data/GFS_typedefs.meta

```diff
@@ -2236,6 +2236,46 @@
   type = real
   kind = kind_phys
   active = (flag_for_stochastic_skeb_option)
+[spp_wts_pbl]
+  standard_name = weights_for_stochastic_spp_pbl_perturbation
```

Comment:
@grantfirl Can you please check the standard names for the newly defined variables? Do they match the standard name rules defined in https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst ?